scilpy/image/volume_operations.py

```diff
@@ -641,3 +641,36 @@ def merge_metrics(*arrays, beta=1.0):
     boosted_mean = geometric_mean ** beta
 
     return ma.filled(boosted_mean, fill_value=np.nan)
+
+
+def dilate_mask(mask, mask_shape, vox_size, radius):
+    """
+    Dilate the foreground in a binary mask according to a radius (in mm)
+
+    toDo: in scil_volume_math, the DILATION option calls binary_dilation from
+     scipy.ndimage. Couldn't we use the same??
```

Comment:
Reformulate the todo so it sounds a bit less dramatic

Reply:
Oups. That's actually already done. Will remove the todo.